portfolio.html

```diff
@@ -6,21 +6,25 @@
 		<title>My Portfolio</title>
 		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
 	</head>	
+	
 	<body>
+		
 		<nav class="nav_bar">
 			<div><label class="label_karen_schulz" for="nav_bar">Karen Schulz</label></div>
 			<div><a href="#">Home</a>
 			<a href="#">Contact</a></div>
 		</nav>
-		<div class="welcome"><img class="mypicture" src="images/profile_pic.png" alt="my_profile_picture">
-			<section>
-			<div class="heading"><h2>Welcome to my Portfolio</h2></div>
-			<div class="portfolio1"><img class="first_work" src="images/green_back.png" alt="my_first_work"><p>I love to do illustrations, check my creations</p>
-			<div class="portfolio2"><img class="second_work" src="images/eating.png" alt="my_second_work"><p>I love html and CSS</p>
-			<div class="portfolio3"><img class="third_work" src="images/full-pouch.png" alt="my_first_work"><p>I've written two children's books</p>>
-		<div><a href="#">Home</a>
```

Comment:
I think this <div> needs to be a </div>

Reply:
I made the links... I guess that is what you meant?